app/controllers/application_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApplicationController < ActionController::Base
+  # protect_from_forgery with: :exception
+  # # no matter which route we log in, we will be redirected
+  # # to the sign in page or sign up page first
+  # before_action :authenticate_user!
+  # before_action :update_allowed_parameters, if: :devise_controller?
+
+  # protected
+
+  # def update_allowed_parameters
+  #   devise_parameter_sanitizer.permit(:sign_up) do |u|
+  #     u.permit(:name, :email, :password, :password_confirmation)
+  #   end
+  #   devise_parameter_sanitizer.permit(:account_update) do |u|
+  #     u.permit(:name, :email, :password, :password_confirmation, :current_password)
+  #   end
+  # end
```

Comment:
- You have done well so far, but please restore back this piece of code, having configured paramaters/strong parameters help to ensure that the database is protected against malicious attacks. Please fix this 👍